tools/train_net.py

```diff
@@ -61,6 +64,13 @@ def train(cfg, local_rank, distributed):
     )
 
     checkpoint_period = cfg.SOLVER.CHECKPOINT_PERIOD
+    if use_tensorboard:
+        meters = TensorboardLogger(
+            log_dir=cfg.TENSORBOARD_EXPERIMENT,
```

Comment:
If you don't mind the question: why did you choose to use another directory instead of using something like `os.path.join(cfg.OUTPUT_DIR, "tensorboard")`?

Reply:
because you may want to store all your tensorboard logs on the same directory.

Reply:
Sorry, I forgot to add a "as the default" to my question. I would expect that the default is to save all output data, including tensorboard-related files in the `OUTPUT_DIR`. Maybe this is only me, idk.